### PR TITLE
187 tutorial basic settings

### PR DESCRIPTION
**Pull request by jalew188**

### CODE REVIEW

File: docs/tutorials/tutorial_basic_definations.ipynb
Comment (GeorgWa) [inline docs/tutorials/tutorial_basic_definations.ipynb:1]: Typo: Definition

File: docs/tutorials/tutorial_dataframe_structures.ipynb
Comment (GeorgWa) [inline docs/tutorials/tutorial_dataframe_structures.ipynb:1]: Maybe mention briefly that this is the "main" format for dense, annotated library and there is also speclib flat.

**Review by GeorgWa — APPROVED** [review]
LGTM

File: docs/tutorials/tutorial_dataframe_structures.ipynb
Comment (jalew188) [reply docs/tutorials/tutorial_dataframe_structures.ipynb:1]: Yes, I should also introduce flat here!

**Pull request merged**